test/contrasts.jl

```diff
@@ -81,6 +81,17 @@
                                 1 -1 -1
                                 1  0  1]
     @test coefnames(mf) == ["(Intercept)"; "x: c"; "x: b"]
+    
+    # Ordinal factors
+    
+    data = DataFrame(x = levels!(categorical(['A', 'B', 'C', 'C'],
+                                             ordered = true),
```

Comment:
Better not set `ordered=true`, as the behavior should be the same for unordered arrays. Then also change "ordinal" to "categorical" above.